app/services/UserService.py

```diff
@@ -31,3 +31,21 @@ async def modify_user(user_id: int, model: PydanticUserModify):
 
     except ValueError as e:
         raise HTTPException(status_code=409, detail=str(e)) from e
+
+async def get_all_users():
+    """
+    Retrieves all users.
+    """
+    users = await UserInDB.all()
+    return [PydanticUserResponse(**user.__dict__) for user in users]
+
+
+async def get_user_by_id(user_id: int):
```

Comment:
Please specify the return type on the function's prototype.

Reply:
`async def get_user_by_id(user_id: int) -> PydanticUserResponse:`